package.json

```diff
@@ -20,9 +21,17 @@
   },
   "homepage": "https://github.com/IgnacioBarocchi/HumbleScraper#readme",
   "dependencies": {
-    "node-fetch": "^2.6.0"
+    "@google-cloud/firestore": "^4.0.0",
+    "@types/firebase": "^3.2.1",
+    "@types/utf8": "^2.1.6",
+    "cheerio": "^1.0.0-rc.3",
+    "fetch-cheerio-object": "^1.3.0",
+    "firebase": "^7.15.5",
+    "node-fetch": "^2.6.0",
+    "utf8": "^3.0.0"
   },
   "devDependencies": {
+    "@types/cheerio": "^0.22.18",
```

Comment:
"@types/Cheerio"
"@types/node"
"@types/node-fetch"
should be regular dependencies (not just devDependencies